tests/integration/conftest.py

```diff
@@ -887,3 +887,10 @@ def truncate_log_files():
 
     for log_file in log_files:
         truncate_file(log_file)
+
+
+@pytest.fixture(scope='function')
+def stop_modules_function():
```

Comment:
Wazuh modulesd is stopped only in the teardown. Give a name that can understand that

Reply:
https://github.com/wazuh/wazuh-qa/pull/2547